nss/Survey_NSS/02_Table of Means NSS 70.do

```diff
@@ -105,11 +113,12 @@ keep if head_age >= 24
 keep if total_debt > 0 
 
 *generate share of gold loan on total liability
-gen double gold_loan_share = gold_loan/total_debt
-gen double unsecure_share = unsecure_loan/total_debt
-gen mrtg_loan_share = mrtg_loan/total_debt
-
+gen double gold_loan_share = total_gold_loan/total_debt
+gen double unsecure_share = total_unsecure_loan/total_debt
+gen mrtg_loan_share = total_mrtg_loan/total_debt
+*Need to match 55%, 23% and 5%
 sum gold_loan_share unsecure_share mrtg_loan_share [aw = pwgt] //gold 6.8%, unsecured loan 56%, mortgage loan 23%
 sum gold_loan_share unsecure_share mrtg_loan_share [aw = hhwgt]
+sum gold_loan_share unsecure_share mrtg_loan_share // Need to use unweighted to match paper 
```

Comment:
The figures in the paper are weighted with the population weight. 
pp 15: "In Panel A, we compute averages across households using population weights, as indicated in each survey"